advanced/dapps/react-dapp-v2/next.config.js

```diff
@@ -2,7 +2,6 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
-  distDir: "build",
```

Comment:
this change would break current vercel setup

Reply:
done